src/test.mediacreation.cpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+// Include all headers to test they are where we expect and can be
+// compiled.
+#include <openassetio/TraitsData.hpp>
+
+#include <openassetio_mediacreation/openassetio_mediacreation.hpp>
```

Comment:
[fix] Be good to add _all_ the headers here. A bit of a maintenance burden I guess, but if that's the argument, then we may as well only include the top-level header.

Reply:
I'm not sure what you mean, these are all the mediacreation headers. 11 headers, 11 includes.

```
.
└── openassetio_mediacreation
    ├── openassetio_mediacreation.hpp
    └── traits
        ├── content
        │   └── LocatableContentTrait.hpp
        ├── content.hpp
        ├── managementPolicy
        │   ├── ManagedTrait.hpp
        │   └── ResolvesFutureEntitiesTrait.hpp
        ├── managementPolicy.hpp
        ├── timeline
        │   ├── ClipTrait.hpp
        │   ├── TimelineTrait.hpp
        │   └── TrackTrait.hpp
        ├── timeline.hpp
        └── traits.hpp
 ```
 
Am I missing something? Are you objecting to having the hoisting headers in place as well as the final ones? I thought the point was just to make sure the #include works.

Reply:
Apologies, I got my wires crossed, thinking we were at least missing specifications, but there aren't any.  Plus now we have #8.